docs/changelog/90116.yaml

```diff
@@ -2,4 +2,34 @@ pr: 90116
 summary: "Release time-series (TSDB) functionality"
 area: "TSDB"
 type: feature
-issues: []
+issues:
+  - 74660
+highlight:
+  title: Release time-series (TSDB) functionality
```

Comment:
Is naming the feature TSDS in line with all stakeholders (such as product and marketing)? I am fine with adopting this name, as long as everyone is onboard.

Reply:
I confirmed that the naming is okay. When James worked on the initial docs he indicated that TSDS was the prefered name: `Rather than publicly call it TSDB (which can be confusing), we’ve decided to call the feature time series data stream (TSDS)`

I also discussed the naming with Nik E to make sure it's okay. Here's the [main PR](https://github.com/elastic/elasticsearch/pull/86905), just for reference.

Reply:
That's great, thanks a lot for the explanation.

cc @giladgal